lib/alert.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+
+import pytz
+
+from lib.alert_date import AlertDate
+
+
+class Alert:
+    def __init__(self, dict_):
+        for key, value in dict_.items():
+            setattr(self, key, value)
+
+        self.sent = AlertDate(self.sent)
+        self.expires = AlertDate(self.expires)
+
+    @property
+    def is_current(self):
+        now = datetime.now(pytz.utc)
+
+        if self.expires.as_utc_datetime <= now:
+            return False
+        return True
```

Comment:
And you could use `SerialisedModelCollection` to avoid manually appending to the list in `build.py`

```suggestion
        return True
        
        
class Alerts(SerialisedModelCollection):
    model = Alert        
```

Example usage:
```python
alerts = Alerts(data['alerts'])
```

Reply:
I was in two minds about this, but then realised a few more benefits:

- We can use it to do the YAML loading
- We can use it to encapsulate the `last_updated_at` conversion
- We can use it to encapsulate filtering current alerts

I've made your suggested change and the `last_updated_at` one in a new commit, and added a further one to extract the current alert filter.